### PR TITLE
HTML Links and Anchors Blog Posts

### PR DESCRIPTION
**Pull request by iamanolive**
## Related Issue
Solves issue #104 (Links and Anchors)

## Description
Wrote blog posts for the following topics:
1. Links Attributes
2. Creating Hyperlinks

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): Content Addition

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: docs/html/links-and-anchors/creating-hyperlinks.md [hunk 1]
```diff
@@ -1,8 +1,196 @@
...
+title: Mastering the Art of Creating Hyperlinks in HTML
 sidebar_label: Creating Hyperlinks
 sidebar_position: 1
 tags: [html, web-development, hyperlinks, links]
 description: In this tutorial, you will learn how to create hyperlinks in HTML. Hyperlinks are used to link one web page to another, or to link to a specific section within the same web page.
----
\ No newline at end of file
+---
+
+# Mastering the Art of Creating Hyperlinks in HTML
```
Comment (Ajay-Dhangar) [inline docs/html/links-and-anchors/creating-hyperlinks.md:10]: remove it

File: docs/html/links-and-anchors/link-attributes.md [hunk 2]
```diff
@@ -1,8 +1,249 @@
...
+
+## In Conclusion
+
+Link attributes in HTML are powerful tools that can enhance the functionality, accessibility, and user experience of your web pages. By mastering these attributes, you can create links that are not only functional but also engaging and user-friendly.
+
+So go ahead, experiment with link attributes in your next project. Use them to create navigation menus, downloadable links, and more. Your users will appreciate the improved navigation and interactivity.
+
+Happy coding!
+
+## Signing Off
```
Comment (Ajay-Dhangar) [inline docs/html/links-and-anchors/link-attributes.md:246]: remove line 243 to 249

File: docs/html/lists/definition-lists.md [hunk 3]
```diff
@@ -145,5 +145,5 @@ Happy coding!
 
 ## Signing Off
```
Comment (Ajay-Dhangar) [inline docs/html/lists/definition-lists.md:146]: remove line 145 to 149

File: docs/html/links-and-anchors/creating-hyperlinks.md [hunk 4]
```diff
@@ -1,8 +1,196 @@
...
+```html
+<a href="https://www.example.com">Tap-friendly Link</a>
+```
+
+## In Conclusion
+
+Creating hyperlinks in HTML is a fundamental skill that every web developer should master. Hyperlinks connect the vast web of information and make navigation intuitive and seamless for users. Whether you're linking to external websites, internal sections, or downloadable documents, the `<a>` tag is your go-to tool.
+
+So go ahead, experiment with hyperlinks in your next project. Use them to create navigation menus, link collections, and more. Your users will appreciate the improved navigation and interactivity.
+
```
Comment (Ajay-Dhangar) [inline docs/html/links-and-anchors/creating-hyperlinks.md:190]: remove line 190 to 196

File: docs/html/lists/ordered-lists.md [hunk 5]
```diff
@@ -161,5 +161,5 @@ Happy coding!
 
 ## Signing Off
```
Comment (Ajay-Dhangar) [inline docs/html/lists/ordered-lists.md:162]: remove line 161 to 165

File: docs/html/lists/list-item-attributes.md [hunk 6]
```diff
@@ -141,5 +141,5 @@ Happy coding!
 
 ## Signing Off
```
Comment (Ajay-Dhangar) [inline docs/html/lists/list-item-attributes.md:143]: remove line 141 to 145

File: docs/html/lists/unordered-lists.md [hunk 7]
```diff
@@ -125,5 +125,5 @@ Happy coding!
 
 ## Signing Off
```
Comment (Ajay-Dhangar) [inline docs/html/lists/unordered-lists.md:126]: remove line 125 to 129

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Please remove some code and want to add something to the page, like an author component, let me know. I can create the component for you, so don't worry.

**Comment by iamanolive** [comment]
hi @Ajay-Dhangar i made the changes you mentioned. could you also please add an author component? and let me know if there's anything else i need to edit.

**Pull request merged**